src/Filter/ChoiceFilter.php

```diff
@@ -66,16 +69,20 @@ public function filter(ProxyQueryInterface $proxyQuery, $alias, $field, $data)
      */
     public function getDefaultOptions()
     {
-        return [];
+        return [
+            'operator_type' => EqualOperatorType::class,
+            'operator_options' => [],
+        ];
     }
 
     /**
      * {@inheritdoc}
      */
     public function getRenderSettings()
     {
-        return ['sonata_type_filter_default', [
-            'operator_type' => 'sonata_type_equal',
+        return [DefaultType::class, [
+            'operator_type' => $this->getOption('operator_type'),
```

Comment:
This is a new feature in DoctrineORM, the ability to override the operator_type (to use an hiddenType for example).